windows-observ-lib/panels.libsonnet

```diff
@@ -449,5 +449,331 @@ local utils = commonlib.utils;
                               targets=[t.networkInPacketsPerSec, t.networkOutPacketsPerSec]
                             )
                             + commonlib.panels.network.timeSeries.traffic.withNegateOutPackets(),
+
+      //	alertsPanel: g.panel.alertList.new(
```

Comment:
why are these commented out?

Reply:
This was my attempt at getting the alerts panel working previous to knowing how, since there is no alerts panel in the common lib. Had this commented out till i had more direction from Vitaly, Got it figured out now!